server.js

```diff
@@ -7,11 +7,11 @@ const lobars = require('lobars')
 // Middleware
 const hbs = require('express-hbs')
 const slashes = require('connect-slashes')
-const browsersync = require('./middleware/browsersync')()
+const browsersync = require('./middleware/browsersync')
```

Comment:
looks like some unrelated changes ended up in this file?

Reply:
Yep. I just edit this for consistency in the `require`s. Remove `()` at this line and add to line 55.